src/lib/compress.c

```diff
@@ -75,6 +75,7 @@ __RCSID("$NetBSD: compress.c,v 1.23 2012/03/05 02:20:18 christos Exp $");
 #include <string.h>
 #include <stdlib.h>
 
+#include "../common/constants.h"
```

Comment:
Yeah, I fixed this by moving constants.h and utils.h to the include. Since we'll need multiple include headers sooner or later anyway we need also to change build stuff (Makefiles?) so we'll put rnp/ folder with contents to include, not the single header. Don't know how to do that yet.